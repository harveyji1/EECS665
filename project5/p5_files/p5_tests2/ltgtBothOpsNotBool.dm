a : int;
b : int;

main : () int {
	c : int;
	c = a < b;
    c = a > b;
    c = a <= b;
    c = a >= b;
}