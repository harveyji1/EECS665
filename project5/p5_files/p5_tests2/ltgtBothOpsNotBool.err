FATAL [6,2]-[6,11]: Invalid assignment operation
FATAL [9,5]-[9,15]: Invalid assignment operation
Type Analysis Failed
