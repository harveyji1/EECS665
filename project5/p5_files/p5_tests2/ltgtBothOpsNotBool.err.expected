FATAL [6,2]-[6,11]: Invalid assignment operation
FATAL [7,5]-[7,14]: Invalid assignment operation
FATAL [8,5]-[8,15]: Invalid assignment operation
FATAL [9,5]-[9,15]: Invalid assignment operation
Type Analysis Failed
